packages/auth-js/src/exception/exception-stack.ts

```diff
@@ -0,0 +1,42 @@
+/**
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+import { AsgardeoAuthException } from "./exception";
+import { AsgardeoAuthNetworkException } from "./network-exception";
+
+export class AsgardeoAuthExceptionStack extends Error {
```

Comment:
Any reason for having a separate exception for the stack? Can't we have the stack in `AsgardeoAuthException` as a constructor arg?

Reply:
Fixed by 55d27d4